src/System.Runtime.Extensions/tests/System/Net/WebUtility.cs

```diff
@@ -63,33 +134,47 @@ public static void UrlDecode(string encodedValue, string expected)
 
         public static IEnumerable<object[]> UrlEncode_TestData()
         {
-            // Recent change brings function inline with RFC 3986 to return hex-encoded chars in uppercase
-            yield return new object[] { "/\\\"\tHello! \u2665?/\\\"\tWorld! \u2665?\u2665", "%2F%5C%22%09Hello!+%E2%99%A5%3F%2F%5C%22%09World!+%E2%99%A5%3F%E2%99%A5" };
-            yield return new object[] { "'", "%27" };
+            foreach (var tuple in UrlEncode_SharedTestData())
+                yield return new object[] { tuple.Item1, tuple.Item2 };
             yield return new object[] { null, null };
         }
 
         [Theory]
         [MemberData(nameof(UrlEncode_TestData))]
+        // [ActiveIssue(7166)]
         public static void UrlEncode(string value, string expected)
         {
             Assert.Equal(expected, WebUtility.UrlEncode(value));
         }
 
         [Theory]
-        [InlineData("'")]
-        [InlineData("http://www.microsoft.com")]
-        [InlineData("/\\\"\tHello! \u2665?/\\\"\tWorld! \u2665?\u2665")]
+        [MemberData(nameof(UrlEncodeDecode_Roundtrip_SharedTestData))]
+        // [ActiveIssue(7166)]
```

Comment:
I left it as a note to emphasize the fact that some tests were commented out, but I didn't want to disable all of the `UrlEncode`/round-trip tests. Will remove the comments.